app/grants/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class GrantsConfig(AppConfig):
+    name = 'grants'
```

Comment:
We should add `verbose_name` here.  Also, we should rename this whole app, directories, and refs to `grant` vs `grants`